highfidelity/hifi
# Fix opened running scripts

check case: https://highfidelity.fogbugz.com/f/cases/5711/Switching-to-HMD-with-Running-Scripts-open-causes-floating-dialog

Test steps:
- In desktop mode, open Running Scripts dialog
- Switch to tablet mode
- Make sure desktop's Running Scripts dialog is closed
- Make sure Tablet opened with Running Scripts dialog
